modules/index-service/src/main/java/org/opencastproject/index/service/impl/index/event/Event.java

```diff
@@ -1158,16 +1052,7 @@ private void updateEventStatus() {
       return;
     }
 
-    if (BooleanUtils.isTrue(getOptedOut())) {
-      eventStatus = "EVENTS.EVENTS.STATUS.OPTEDOUT";
-      return;
-    }
-
-    if (StringUtils.isNotBlank(getSchedulingStatus())) {
```

Comment:
@lkiesow Should now be covered by https://github.com/opencast/opencast/pull/751/files#diff-5d605680ba982aebd6f2623a060d7e3eR1050